src/analyzelog.cpp

```diff
@@ -16,14 +16,15 @@
 #include "src/simbox.h"
 #include "src/io.h"
 
-Analyzelog::Analyzelog(const std::vector<NRLib::Well *>                        & wells,
-                       const std::map<std::string, BlockedLogsCommon *>        & mapped_blocked_logs,
-                       const std::vector<std::vector<NRLib::Grid<float> *> >   & background,
-                       const std::vector<Simbox *>                             & interval_simboxes,
-                       double                                                    dz_min,
-                       const ModelSettings                                     * model_settings,
-                       bool                                                      multi_zone_available,
-                       std::string                                             & err_txt):
+Analyzelog::Analyzelog(const std::vector<NRLib::Well *>                      & wells,
+                       const std::map<std::string, BlockedLogsCommon *>      & mapped_blocked_logs,
+                       const std::vector<std::vector<NRLib::Grid<float> *> > & background,
+                       const std::vector<Simbox *>                           & background_simboxes, //Simboxes per interval, used for subtracting background models if we use multiinterval
+                       const Simbox                                          * estimation_simbox, //Simbox where correlations are estimated
```

Comment:
Endring er at vi sender inn 1 simbox for estimering, også i multisone med det tilfelle hvor det ikke er nok data til å estimere i sone og vi må estimere over alle soner.
Jeg fikk ikke fjernet alle sone-simboxer, fordi man trekker fra bakgrunnsmodellen per sone og da bruker man simboxen til den sonen. Men jeg kalte de 'background_simboxes' for å skulle fra 'estimation_simbox'.